src/Utils/CategoryDropdown.js

```diff
@@ -16,20 +19,53 @@ function CategoryDropdown({ dropdownVendorCategories, name, withLabel, ariaLabel
 
     if (!item) return option;
 
-    return <OptionSegment searchTerm={searchTerm}>{item.value}</OptionSegment>;
-  }, [dropdownVendorCategories]);
+    const translationKey = VENDOR_CATEGORIES[item.value];
+
+    return (
+      <OptionSegment searchTerm={searchTerm}>
+        {translationKey
```

Comment:
this code repeats a lot in this PR, maybe it makes sense to leverage this to a hook